app/src/main/kotlin/com/simplemobiletools/flashlight/activities/WidgetBrightDisplayConfigureActivity.kt

```diff
@@ -3,48 +3,61 @@ package com.simplemobiletools.flashlight.activities
 import android.app.Activity
 import android.appwidget.AppWidgetManager
 import android.content.Intent
-import android.content.res.ColorStateList
-import android.graphics.Color
 import android.os.Bundle
-import android.widget.SeekBar
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.viewModels
+import androidx.compose.runtime.getValue
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.simplemobiletools.commons.compose.extensions.enableEdgeToEdgeSimple
+import com.simplemobiletools.commons.compose.theme.AppThemeSurface
 import com.simplemobiletools.commons.dialogs.ColorPickerDialog
 import com.simplemobiletools.commons.dialogs.FeatureLockedDialog
-import com.simplemobiletools.commons.extensions.*
+import com.simplemobiletools.commons.extensions.isOrWasThankYouInstalled
 import com.simplemobiletools.commons.helpers.IS_CUSTOMIZING_COLORS
 import com.simplemobiletools.flashlight.R
-import com.simplemobiletools.flashlight.databinding.WidgetBrightDisplayConfigBinding
+import com.simplemobiletools.flashlight.activities.viewmodel.WidgetConfigureViewModel
 import com.simplemobiletools.flashlight.extensions.config
 import com.simplemobiletools.flashlight.helpers.MyWidgetBrightDisplayProvider
+import com.simplemobiletools.flashlight.screens.WidgetConfigureScreen
 
-class WidgetBrightDisplayConfigureActivity : SimpleActivity() {
-    private val binding by viewBinding(WidgetBrightDisplayConfigBinding::inflate)
+class WidgetBrightDisplayConfigureActivity : ComponentActivity() {
+    private val viewModel by viewModels<WidgetConfigureViewModel>()
 
-    private var mWidgetAlpha = 0f
-    private var mWidgetId = 0
-    private var mWidgetColor = 0
-    private var mWidgetColorWithoutTransparency = 0
     private var mFeatureLockedDialog: FeatureLockedDialog? = null
```

Comment:
This one should be migrated to Compose too `FeatureLockedAlertDialog`

Reply:
Right, missed that one completely
Should be quick to update though

Reply:
You have it in two places don't forget.